pkg/controller/client.go

```diff
@@ -48,14 +48,14 @@ type XdsClient struct {
 	xdsConfig          *config.XdsConfig
 }
 
-func NewXdsClient(mode string, bpfWorkload *bpf.BpfKmeshWorkload) *XdsClient {
+func NewXdsClient(mode string, bpfWorkload *bpf.BpfKmeshWorkload, enableAccesslog bool) *XdsClient {
 	client := &XdsClient{
 		mode:      mode,
 		xdsConfig: config.GetConfig(mode),
 	}
 
 	if mode == constants.WorkloadMode {
-		client.WorkloadController = workload.NewController(bpfWorkload)
+		client.WorkloadController = workload.NewController(bpfWorkload, enableAccesslog)
```

Comment:
Access log is only supported in workload mode?

Reply:
Yes